code/datums/components/tackle.dm

```diff
@@ -436,6 +436,9 @@
 	var/obj/item/organ/external/wings/sacker_wing = sacker.get_organ_slot(ORGAN_SLOT_EXTERNAL_WINGS)
 	if(sacker_wing)
 		attack_mod += 2
+	var/obj/item/organ/internal/ears/cat/tajaran/sacker_tajaran_ears = sacker.get_organ_slot(ORGAN_SLOT_EARS)//Bubberstation Edit start- Cat's are particularly good at pouncing
+	if(HAS_TRAIT(sacker, TRAIT_CATLIKE_GRACE) && sacker_tajaran_ears)
+		attack_mod += 2//Bubberstation Edit finish
```

Comment:
You're more likely to lack the ears than the trait. Also needs to typecheck if you want it tied to the organ type  specifically.

```suggestion
	var/obj/item/organ/internal/ears/cat/tajaran/sacker_tajaran_ears = sacker.get_organ_slot(ORGAN_SLOT_EARS)//Bubberstation Edit start- Cat's are particularly good at pouncing
	if(istype(sacker_tajaran_ears) && HAS_TRAIT(sacker, TRAIT_CATLIKE_GRACE))
		attack_mod += 2//Bubberstation Edit finish
```

Reply:
Yeah, Fair point. Thank you.